WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/PrepublishingHomeItemUiState.kt

```diff
@@ -18,50 +21,63 @@ sealed class PrepublishingHomeItemUiState {
 
     sealed class ButtonUiState(
-        val buttonText: UiString.UiStringRes,
+        val buttonText: UiStringRes,
         val publishPost: PublishPost
     ) : PrepublishingHomeItemUiState() {
         open val onButtonClicked: ((PublishPost) -> Unit)? = null
 
         data class PublishButtonUiState(override val onButtonClicked: (PublishPost) -> Unit) : ButtonUiState(
-            UiString.UiStringRes(R.string.prepublishing_nudges_home_publish_button),
+            UiStringRes(R.string.prepublishing_nudges_home_publish_button),
             true
         )
 
         data class ScheduleButtonUiState(override val onButtonClicked: (PublishPost) -> Unit) : ButtonUiState(
-            UiString.UiStringRes(R.string.prepublishing_nudges_home_schedule_button),
+            UiStringRes(R.string.prepublishing_nudges_home_schedule_button),
             false
         )
 
         data class UpdateButtonUiState(override val onButtonClicked: (PublishPost) -> Unit) : ButtonUiState(
-            UiString.UiStringRes(R.string.prepublishing_nudges_home_update_button),
+            UiStringRes(R.string.prepublishing_nudges_home_update_button),
             false
         )
 
         data class SubmitButtonUiState(override val onButtonClicked: (PublishPost) -> Unit) : ButtonUiState(
-            UiString.UiStringRes(R.string.prepublishing_nudges_home_submit_button),
+            UiStringRes(R.string.prepublishing_nudges_home_submit_button),
             true
         )
 
         data class SaveButtonUiState(override val onButtonClicked: (PublishPost) -> Unit) : ButtonUiState(
-            UiString.UiStringRes(R.string.prepublishing_nudges_home_save_button),
+            UiStringRes(R.string.prepublishing_nudges_home_save_button),
             false
         )
     }
 
-    enum class ActionType(val textRes: UiString.UiStringRes) {
-        PUBLISH(UiString.UiStringRes(R.string.prepublishing_nudges_publish_action)),
-        TAGS(UiString.UiStringRes(R.string.prepublishing_nudges_tags_action)),
-        CATEGORIES(UiString.UiStringRes(R.string.prepublishing_nudges_categories_action)),
-        ADD_CATEGORY(UiString.UiStringRes(R.string.prepublishing_nudges_categories_action))
+    data class SocialUiState(
+        val title: UiString,
+        val description: UiString,
+        val isLowOnShares: Boolean,
+        val connectionIcons: List<ConnectionIcon>,
+        val onItemClicked: (() -> Unit)?
+    ) : PrepublishingHomeItemUiState() {
+        data class ConnectionIcon(
+            @DrawableRes val iconRes: Int,
+            val isEnabled: Boolean = true
+        )
+    }
+
+    enum class ActionType(val textRes: UiStringRes) {
+        PUBLISH(UiStringRes(R.string.prepublishing_nudges_publish_action)),
+        TAGS(UiStringRes(R.string.prepublishing_nudges_tags_action)),
+        CATEGORIES(UiStringRes(R.string.prepublishing_nudges_categories_action)),
+        ADD_CATEGORY(UiStringRes(R.string.prepublishing_nudges_categories_action))
```

Comment:
❓ `ADD_CATEGORY` has the same `textRes` as `CATEGORIES` (`R.string.prepublishing_nudges_categories_action`). Is this expected?

Reply:
I believe it is. Bear in mind I didn't change nor add this but looking at the existing code it seems to make sense, as this is just being used to navigate to the appropriate fragment and this string is not being used in the UI for the `ADD_CATEGORY` case.